using-mobile-extensions/adobe-analytics-edge/analytics-edge-api-reference.md

```diff
@@ -0,0 +1,126 @@
+# Analytics Edge API reference
+
+{% hint style="warning" %}
+
+The AEP Analytics Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more.
+
+{% endhint %}
+
+## trackAction
```

Comment:
do we still need those APIs to be documented here since they have already been documented in Core?

Reply:
i think we can remove the track API from this page and just link to the Core docs. I'll leave the information about the unsupported API.